Harvest.Web/Startup.cs

```diff
@@ -52,7 +55,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllersWithViews(options =>
             {
                 options.Filters.Add<SerilogControllerActionFilter>();
-            }).AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new NetTopologySuite.IO.Converters.GeoJsonConverterFactory()));
+            }).AddJsonOptions(options =>

```

Comment:
@sprucely If we have these options here in startup, do we still need them to be specified when we deserialize?

Reply:
I believe so, because these options apply only to the automatic serialization/deserialization performed by aspnetcore in the request/response pipeline.